### ISSUE TITLE
Consider improvements to identify the ultimate recipient

### ISSUE THREAD

**Issue by JonellaCulmer** [fecgov/fec-cms#6199]
###  What we're after
There is some confusion on transactions where the ultimate recipient and a payee are different. Some instances include in-kind contributions, when reported correctly. We need to explore opportunities to clarify the differences between these two different types of "recipients" 

Examples: 
- https://docquery.fec.gov/cgi-bin/fecimg/?202301259574959243
- https://www.fec.gov/data/disbursements/?cycle=2022&data_type=processed&recipient_name=IAO+Property+Holdings+LLC&two_year_transaction_period=2022&disbursement_description=in+kind

### Related issues
- https://github.com/fecgov/fec-cms/issues/6172

### Action items
- [ ] Create design solutions
- [ ] Propose designs with content and PMs
- [ ] Finalize designs and add to [dev ticket](https://github.com/fecgov/fec-cms/issues/6172)

### Completion criteria
- [ ] Designs created that differentiate a payee and ultimate recipient of funds/benefit

**Comment by JonellaCulmer** [comment]
@PaulClark2 @patphongs @cnlucas @hcaofec Here's a mockup of what small change we can potentially make to add clarity to the details panel of the disbursement table, when there are two different names present for in-kinds. Wondering if we can hide this new line if there is no `beneficiary_committee_name` which is where the candidate committee seems to be pulled from. I would not change the columns in the table. 

![Image](https://github.com/fecgov/fec-cms/assets/31663028/6090ee06-732b-4ef9-a53f-aecb6576c0a9)

**Comment by PaulClark2** [comment]
Thanks @JonellaCulmer. 

For the Pence in-kind example, will the data table still say "IN-KIND; C00658401; GREG PENCE FOR CONGRESS; BASEMENT " in the description field? 
![image](https://github.com/fecgov/fec-cms/assets/11817734/1c7ec3b4-86ab-4931-a62c-90f14b16c4ec)


![image](https://github.com/fecgov/fec-cms/assets/11817734/570403ca-7f9e-45ae-b8d6-e9ab17385c98)

**Comment by PaulClark2** [comment]
Does this also address/resolve the RNC/Warren Stevens/WinRed issue?

![image](https://github.com/fecgov/fec-cms/assets/11817734/b3db9ab3-7f8f-4ae4-881e-33a610f990a6)

**Comment by JonellaCulmer** [comment]
@PaulClark2 Yes, beneficiary_committee_name is in the datatable download already. But I planned to just keep the beneficiary in the recipient column which keeps it simple for both "regular" contributions and in-kinds. That's why I nested the beneficiary in the recipient section. Let me know if you think that adds confusion for you rather than clarity.

**Comment by JonellaCulmer** [comment]
Closing in favor of implementation ticket https://github.com/fecgov/fec-cms/issues/6172
